frontend/turbo.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Same question re: defaults vs. explicit choices

Reply:
These are defaults with the exception of line 8 where I added a `dist` output for the design system.